plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -52,12 +52,13 @@ func (nodes ClientNodeInfo) GetNodeInfo(nodeID string) (*api.Node, error) {
 
 func isVolumeConflict(volume api.Volume, pod *api.Pod) bool {
```

Comment:
Medium-to-long term, anything related to specific volume implementations should move to pkg/volume and be part of the plugin interface.